src/main/java/com/cegeka/academy/web/rest/InvitationController.java

```diff
@@ -0,0 +1,45 @@
+package com.cegeka.academy.web.rest;
+
+import com.cegeka.academy.domain.Invitation;
+import com.cegeka.academy.service.invitation.InvitationService;
+import com.cegeka.academy.service.invitation.ValidationAccessService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("invitation")
+public class InvitationController {
+
+    private final InvitationService invitationService;
+
+    private final ValidationAccessService validationAccessService;
+
+    @Autowired
+    public InvitationController(InvitationService invitationService, ValidationAccessService validationAccessService) {
+        this.invitationService = invitationService;
+        this.validationAccessService = validationAccessService;
+    }
+
+    @GetMapping("/all")
+    public String getAllInvitations(){
+        return invitationService.getAllInvitations();
+    }
+
+    @PostMapping("/post")
```

Comment:
Request-uri de genul `POST /_entityName_` sunt suficiente ca sa spuna "asta e un save".
Vezi [aici](https://martinfowler.com/articles/richardsonMaturityModel.html) nivelurile de la 0 la 2, nivelul 3 nu intra in discutie.

Request detaliat pentru un _save_:
`
POST http://localhost:8080/invitaion
{
description: "o descriere de invitatie",
status: "IN_PROGRESS"
}
`